projects/RabbitMQ.Client/client/impl/IFrameHandler.cs

```diff
@@ -57,10 +58,12 @@ internal interface IFrameHandler
         ///<summary>Read a frame from the underlying
         ///transport. Returns null if the read operation timed out
```

Comment:
this is not true for the new impl. if this behavior is desired the impl needs to add cancellationtokens to handle timeouts and return null.

Reply:
I'll take a closer look at this. It's not surfaced in any tests it seems.